### ISSUE TITLE
In short, after I get a flowDict, how can I know it's value easily?

### ISSUE THREAD

**Issue by RocStone** [networkx/networkx#2509]
using this
`mincostFlow = nx.max_flow_min_cost(G, 9506, 9505)`

I can get a flowDict of min cost max flow of G, now I want to know it's flow value
I searched the inference of networkx, I found this function
`value = nx.maximum_flow_value(G, 9506, 9505)`

but i think if there exists a function can count the flows of `mincostFlow` should be faster than `nx.maximum_flow_value(G, 9506, 9505)` right?

**In short, after I get a flowDict, how can I know it's value easily?**
**this lib is awesome, thanks for your work, wish you happy every day.**

**Comment by dschult** [comment]
You might get a quicker response from the [mailing list](https://groups.google.com/forum/#!forum/networkx-discuss) as this is for developer issues. 

The [docstring for the function](http://networkx.readthedocs.io/en/stable/reference/generated/networkx.algorithms.flow.max_flow_min_cost.html?highlight=max_flow_min_cost) says that the flow is reported by ```FlowDict[9506][9505]```

I'm going to close this issue, but feel free to post to the mail-list or reopen this issue.
